### PR TITLE
Update node-meetups.md

### PR DESCRIPTION
**Pull request by SofiEstevez**
Added a lot of meetups around the world :)

### CODE REVIEW

File: locale/en/get-involved/node-meetups.md
Comment (ZYSzys) [inline locale/en/get-involved/node-meetups.md]: Some trivial nits: could you please be so kind as to update all info following the format as below :)
```suggestion
- Organizer contact info - sidorares@yandex.com
```

**Review by ZYSzys — APPROVED** [review]
Almost LGTM

**Review by ZYSzys — COMMENTED** [review]
It seems the format should be:

```
### Country(like United States)

#### State(like California)

##### City(like San Francisco)
```


See: https://github.com/nodejs/nodejs.org/pull/1992/files#diff-f20affdaee1114975a78c88cafabd0adR30

File: locale/en/get-involved/node-meetups.md
Comment (willin) [inline locale/en/get-involved/node-meetups.md]: i wonder why this line was deleted?

**Review by willin — APPROVED** [review]

File: locale/en/get-involved/node-meetups.md
Comment (SofiEstevez) [reply locale/en/get-involved/node-meetups.md]: Thought maybe it was obvious but no worries, will add again!

**Comment by ghost** [comment]
@SofiEstevez：Maybe you can solve the conflicting files and push a submit, and we'll merge them.

**Comment by SofiEstevez** [comment]
@Maledong Thank you, I just resolved conflicts :)

**Comment by SofiEstevez** [comment]
@ZYSzys are we good to merge?

File: locale/en/get-involved/node-meetups.md
Comment (ZYSzys) [inline locale/en/get-involved/node-meetups.md]:
```suggestion
### Greece
```

File: locale/en/get-involved/node-meetups.md
Comment (ZYSzys) [inline locale/en/get-involved/node-meetups.md]:
```suggestion
##### Atlanta 
```

**Review by ZYSzys — COMMENTED** [review]
Yes, we'll merge this once suggestions below landed.(if my suggestions below are not wrong) :)

**Comment by ZYSzys** [comment]
Hi @SofiEstevez , I'm a bit of curoious why you delete some `Organizer contact info` 🤔?

**Comment by SofiEstevez** [comment]
@ZYSzys oh sorry, I just realized I prefer to don't assume these people want to share their info.

**Review by ZYSzys — APPROVED** [review]

**Comment by ZYSzys** [comment]
Thank you !👍

**Pull request merged**
